typescript/infra/config/environments/testnet2/middleware/liquidity-layer/verification.json

```diff
@@ -17,6 +17,18 @@
       "address": "0xe589a05be1304b43A6FEb9c5D6a6EEEA35656271",
       "isProxy": false,
       "constructorArguments": ""
+    },
```

Comment:
Wondering why this is here, unless it *does* include a deployment? If it includes a deployment wouldn't we expect all of the addresses to change?